fenix/app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -503,6 +505,21 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun setScrollingBehavior() {
+        if (binding.homeAppBar.layoutParams != null) {
+            val appBarLayoutParams = binding.homeAppBar.layoutParams as CoordinatorLayout.LayoutParams
+            val appBarBehavior = AppBarLayout.Behavior()
+            appBarBehavior.setDragCallback(
+                object : AppBarLayout.Behavior.DragCallback() {
+                    override fun canDrag(appBarLayout: AppBarLayout): Boolean {
+                        return false
+                    }
+                },
+            )
```

Comment:
Interesting approach to disable dragging in all scenarios.

Looking through the initial tickets for the current functionality - https://github.com/mozilla-mobile/fenix/issues/99, https://github.com/mozilla-mobile/fenix/issues/137, https://github.com/mozilla-mobile/fenix/issues/7700 I don't see any discussions or indications about the scenarios in which we'd want the appBar to be draggable or not.

Given that currently the appBar can be dragged only when there aren't enough elements on the screen to overflow it and in this scenario the drag gesture results in the bug we need so solve disabling dragging seems like the sensible choice.